src/main/scala/viper/gobra/frontend/info/implementation/property/Convertibility.scala

```diff
@@ -18,6 +18,7 @@ trait Convertibility extends BaseProperty { this: TypeInfoImpl =>
   } {
     case (Single(lst), Single(rst)) => (lst, rst) match {
       case (left, right) if assignableTo(left, right) => true
+      case (IntT(_), IntT(_)) => true
```

Comment:
I am sure you checked this, but just to make sure: In Go, any integer kind can be converted to every integer kind?

Reply:
The go language spec states that
>  A non-constant value x can be converted to type T in any of these cases:
    x is assignable to T.
    ignoring struct tags (see below), x's type and T have identical underlying types.
    ignoring struct tags (see below), x's type and T are pointer types that are not defined types, and their pointer base types have identical underlying types.
    **x's type and T are both integer or floating point types.**
    x's type and T are both complex types.
    x is an integer or a slice of bytes or runes and T is a string type.
    x is a string and T is a slice of bytes or runes.

In particular, there is no constraint on the types of integers allowed for conversion. One thing that I should note however is that this is valid for a non-constant value x. The well-defidedness of a conversion of a constant expression x to a type T relies on a different criterion however
> A constant value x can be converted to type T if x is representable by a value of T.

Besides,
> **A constant x is representable by a value of type T if one of the following conditions applies:
    x is in the set of values determined by T.**
    (other clauses not relevant for the discussion)

This latter case (for constants) is not implemented in Gobra because `Convertibility` is implemented as a binary relation between types instead of being implemented as a relation between an expression and a type to which the expression is converted to. This simplification is consistent with the one made for the `Assignability` property, which is also a relation between a value x and a type according to the Go lang. spec. but is implemented in Gobra as a `Property[(Type, Type)]`